DQMOffline/Trigger/plugins/NoBPTXMonitor.h

```diff
@@ -0,0 +1,125 @@
+#ifndef NOBPTXMONITOR_H
```

Comment:
hi @jalimena - thanks - all of my requests look to be fixed. I missed two (as numerous PRs have the same patterns..)

1) please add the package name to be part of the include guard here. eg, DQMOFFLINE_TRIGGER_NOBPTXMONITOR_H

2) change nbins to be an unsigned int